strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/AuthorizationConfigController.java

```diff
@@ -192,8 +192,8 @@ public ResponseEntity deleteRole(@ApiParam(value = "The name of the role", requi
     // Assign privileges to the anonymous user
 
     @RequestMapping(value = "anonymous/privileges",
-                    method = RequestMethod.POST,
-                    consumes = MediaType.APPLICATION_JSON_VALUE)
+            method = RequestMethod.POST,
```

Comment:
How about if you move the exclusion above the `.idea/` and then commit just the `.gitignore` and after that commit the exluded xml file?